src/util/Util.js

```diff
@@ -18,12 +18,26 @@ class Util {
     constructor() {
         throw new Error(`The ${this.constructor.name} class may not be instantiated.`);
     }
+    /**
+     * Create a nested directory
+     * @param {string} path 
+     */
+    static async createNestedDirectory(path) {
```

Comment:
If we do want to end up creating the dir beforehand, node v10+ supports a `recursive` bool on mkdir that we can use. I don't think this custom util function is necessary.

Reply:
great!